ui/overview_page.go

```diff
@@ -382,27 +481,65 @@ func (page *overviewPage) endToEndRow(inset layout.Inset, leftLabel, rightLabel
 // syncBoxTitleRow lays out widgets in the title row inside the sync status box.
 func (page *overviewPage) syncBoxTitleRow(inset layout.Inset) {
 	statusTitleLabel := page.theme.Caption(page.text.statusTitle)
-	statusTitleLabel.Color = page.gray
+	statusTitleLabel.Color = page.theme.Color.Text
 	statusLabel := page.theme.Body1(page.text.offlineStatus)
+	page.walletStatusIcon.Color = page.theme.Color.Danger
 	if page.walletInfo.Synced || page.walletInfo.Syncing {
 		statusLabel.Text = page.text.onlineStatus
+		page.walletStatusIcon.Color = page.theme.Color.Success
 	}
-	page.endToEndRow(inset, statusTitleLabel, statusLabel)
+
+	gtx := page.gtx
+	inset.Layout(gtx, func() {
+		layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+			layout.Rigid(func() {
+				statusTitleLabel.Layout(gtx)
+			}),
+			layout.Flexed(1, func() {
+				layout.E.Layout(gtx, func() {
+					layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+						layout.Rigid(func() {
+							layout.Inset{Top: unit.Dp(6), Right: unit.Dp(20)}.Layout(gtx, func() {
+								page.walletStatusIcon.Layout(gtx, unit.Dp(10))
```

Comment:
remove hard coded dimensions. Dimensions defined in dimension.go are standardized to a 5 unit difference. So instead of 6, you can use 5 and instead of 8, you can use 10. You can define a unique dimension that don't follow the standardized unit difference only when it's absolutely necessary.